terraform/README.adoc

```diff
@@ -34,6 +47,9 @@ new nvmf target or what have you.
 So we need a form of infra management but also - configuration management.
 These scripts are intended to do just that.
 
+Currently the scripts don't support anything else than KVM and LXD
+deployments.
```

Comment:
I think this is clearer as "...scripts only support KVM and LXD deployments".

Reply:
agree. thanks!